programs/C/length_string.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+//Created a function to calculate length of string.
+int LengthOfString(char string[50])
```

Comment:
use camel case

Reply:
@divyam3897 Sir, please see if I have done it correctly or not.